tests/unit/versioning/VersionID.js

```diff
@@ -9,83 +10,121 @@ function randkey(length) {
     return key;
 }
 
-describe('test generating versionIds', () => {
-    const count = 1000;
+process.env.VID_CRYPTO_PASSWORD = randkey(64);
```

Comment:
Can we restore the env after tests are run? Given we don't care about modifying the value of `VID_CRYPTO_PASSWORD`, this can be as simple as saving `process.env` in `beforeAll` and restoring it in `afterAll`.